qdrant-landing/content/articles/what-is-quantization.md

```diff
@@ -0,0 +1,242 @@
+
+$$
+\frac{1536 \text{ dimensions}}{8 \text{ bits per byte}} = 192 \text{ bytes}
+$$
+
+
+This leads to a **32x** memory saving.
+
+
+![Binary Quantization example](/articles_data/what-is-vector-quantization/binary-quantization.png)
+
+
+Qdrant automates the Binary Quantization process during indexing. As vectors are added to your collection, each 32-bit floating-point component is converted into a binary value according to the configuration you define. 
+
+Here’s how you can set it up:
+
+```python 
+collection_config = {
+    "vectors": {
+        "size": 1536,
+        "distance": "Cosine"
+    },
+    "quantization_config": {
+        "binary": {
+            "always_ram": True
+        }
+    }
+}
+```
+
+Binary Quantization is by far the quantization method that will give you the most processing **speed gains **when compared to Scalar and Product Quantizations. This is because the binary representation allows the system to use highly optimized CPU instructions, such as [XOR](https://en.wikipedia.org/wiki/XOR_gate#:~:text=XOR%20represents%20the%20inequality%20function,the%20other%20but%20not%20both%22.) and [Popcount](https://en.wikipedia.org/wiki/Hamming_weight), for fast distance computations.
+
+It can speed up search operations by **up to 40x**, depending on the dataset and hardware. Here's the speed comparison of all three methods:
+
+<img src="/articles_data/what-is-vector-quantization/speed-methods.png" alt="Speed by Quantization Method" width="700">
+
+
+But it’s important to note that not all models are equally compatible with Binary Quantization. Some models may suffer a larger loss in accuracy when quantized, so it's recommended to use it on models of **at least 1024 dimensions.** 
+
+The models that have shown the best compatibility with this method include:
+
+* **OpenAI text-embedding-ada-002** (1536 dimensions)
+* **Cohere AI embed-english-v2.0** (4096 dimensions)
+
+They demonstrate minimal accuracy loss while still benefiting from the substantial speed and memory gains.
+
+Even though Binary Quantization is incredibly fast and memory-efficient, the trade-offs are in **precision** and **model compatibility**, and you may need **rescoring** to ensure search quality.
+
+# What is Rescoring?
+
+The quantization strips away some detail from the original vector. However, Qdrant mitigates this with **oversampling**, which retrieves extra vectors during the search and then uses the original values to rescore them, improving the accuracy of the final results.
+
+Here’s how you can enable rescoring to maintain higher accuracy:
+
+```python 
+search_payload = {
+    "query": [0.22, -0.01, -0.98, 0.37, ...],
+    "params": {
+        "quantization": {
+            "rescore": True
+        }
+    },
+    "limit": 10
+}
+```
+
+When you enable rescoring in Qdrant, it refines the top search results by recalculating their similarity with the original floating-point vectors.
```

Comment:
It's an important paragraph that deserves some more attention. Qdrant keeps the original vectors, so there is a way back even if someone enables quantization on a specific collection.

Reply:
Yes, I'm still working on the rescoring and reranking explanations!